Sources/Sentry/SentryAppStateManager.m

```diff
@@ -72,6 +72,17 @@ - (void)removeCurrentAppState
     [self.fileManager deleteAppState];
 }
 
+- (void)updateAppState:(void (^)(SentryAppState *))block
+{
+    @synchronized(self) {
+        SentryAppState *appState = [self.fileManager readAppState];
+        if (nil != appState) {
```

Comment:
What about the first run? Will readAppState create an empty file for us on read?

Reply:
As we currently only use this for creating OOMs and we use it to avoid creating false OOMs, we don't need to worry about this right now, as we only create an OOM if there is an app state.